spec/lib/swagger_finder_spec.rb

```diff
@@ -19,7 +19,7 @@
     subject(:classes) { described_class.new.classes }
 
     it 'finds the classes' do
-      expect(classes.first.to_s.include?('Nudged::V1::Swagger')).to be_truthy
+      expect(classes).to include(Ec::Nudged::V1::Swagger)
```

Comment:
There's a lot of code here -- should there be more specs?

Reply:
swagger_finder is at 100% codecov